Bugzilla/Bug.pm

```diff
@@ -4635,15 +4635,21 @@ sub GetBugActivity {
         my @filtered_added;
         foreach my $value (split /, /, $removed) {
           if (substr($value, 0, length $url_base) eq $url_base) {
-            my ($bug_id) = $value =~ /id=(\d+)$/;
-            next if !Bugzilla->user->can_see_bug($bug_id);
+            my ($bug_id) = $value =~ /id=(\w+)$/;
+            if ($bug_id !~ /^\d+$/) {
+              $bug_id = bug_alias_to_id($bug_id);
```

Comment:
What is a "bug alias"?

Reply:
A bug alias is just a unique text identifier for a bug instead of using the ID. It has to be unique and can be used in places that accept a bug ID. 

Example: https://bugzilla.mozilla.org/show_bug.cgi?id=MediaSession